fenix/app/src/main/java/org/mozilla/fenix/translations/preferences/automatic/AutomaticTranslationOptionsPreference.kt

```diff
@@ -47,6 +47,7 @@ fun AutomaticTranslationOptionsPreference(
                         item.summaryId.first(),
                         stringResource(item.summaryId.last()),
                     ),
+                    maxDescriptionLines = Int.MAX_VALUE,
```

Comment:
`Int.MAX_VALUE` seems a bit large to me, but I see we have [used it once before](https://searchfox.org/mozilla-mobile/search?path=&q=maxDescriptionLines%20%3D) and for other view [items](https://searchfox.org/mozilla-mobile/search?q=Int.MAX_VALUE&path=&case=false&regexp=false).